diff --git i/index.html w/index.html
index 2eefeee..60a30d2 100644
--- i/index.html
+++ w/index.html
@@ -4,21 +4,21 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0" />
-    <title>Cinny</title>
-    <meta name="name" content="Cinny" />
-    <meta name="author" content="Ajay Bura" />
+    <title>Harrier</title>
+    <meta name="name" content="Harrier" />
+    <meta name="author" content="schadin" />
     <meta
       name="description"
       content="A Matrix client where you can enjoy the conversation using simple, elegant and secure interface protected by e2ee with the power of open source."
     />
     <meta
       name="keywords"
-      content="cinny, cinnyapp, cinnychat, matrix, matrix client, matrix.org, element"
+      content="harrier, matrix, matrix client, matrix.org"
     />
 
-    <meta property="og:title" content="Cinny" />
-    <meta property="og:url" content="https://cinny.in" />
-    <meta property="og:image" content="https://cinny.in/assets/favicon-48x48.png" />
+    <meta property="og:title" content="Harrier" />
+    <meta property="og:url" content="https://github.com/schadin/harrier-desktop" />
+    <meta property="og:image" content="./public/res/favicon-48x48.png" />
     <meta
       property="og:description"
       content="A Matrix client where you can enjoy the conversation using simple, elegant and secure interface protected by e2ee with the power of open source."
@@ -29,8 +29,8 @@
 
     <link rel="manifest" href="/manifest.json" />
     <meta name="mobile-web-app-capable" content="yes" />
-    <meta name="application-name" content="Cinny" />
-    <meta name="apple-mobile-web-app-title" content="Cinny" />
+    <meta name="application-name" content="Harrier" />
+    <meta name="apple-mobile-web-app-title" content="Harrier" />
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
 
diff --git i/public/manifest.json w/public/manifest.json
index acdcd00..7821eb8 100644
--- i/public/manifest.json
+++ w/public/manifest.json
@@ -1,7 +1,7 @@
 {
-  "name": "Cinny",
-  "short_name": "Cinny",
-  "description": "Yet another matrix client",
+  "name": "Harrier",
+  "short_name": "Harrier",
+  "description": "Matrix desktop client powered by Cinny",
   "dir": "auto",
   "lang": "en-US",
   "display": "standalone",
diff --git i/src/app/components/splash-screen/SplashScreen.tsx w/src/app/components/splash-screen/SplashScreen.tsx
index 27adadb..5258a66 100644
--- i/src/app/components/splash-screen/SplashScreen.tsx
+++ w/src/app/components/splash-screen/SplashScreen.tsx
@@ -21,7 +21,7 @@ export function SplashScreen({ children }: SplashScreenProps) {
         justifyContent="Center"
       >
         <Text size="H2" align="Center">
-          Cinny
+          Harrier
         </Text>
       </Box>
     </Box>
diff --git i/src/app/features/settings/about/About.tsx w/src/app/features/settings/about/About.tsx
index 7f6d90a..2bcbf7f 100644
--- i/src/app/features/settings/about/About.tsx
+++ w/src/app/features/settings/about/About.tsx
@@ -4,7 +4,7 @@ import { Page, PageContent, PageHeader } from '../../../components/page';
 import { SequenceCard } from '../../../components/sequence-card';
 import { SequenceCardStyle } from '../styles.css';
 import { SettingTile } from '../../../components/setting-tile';
-import CinnySVG from '../../../../../public/res/svg/cinny.svg';
+import HarrierSVG from '../../../../../public/res/svg/harrier.svg';
 import { clearCacheAndReload } from '../../../../client/initMatrix';
 import { useMatrixClient } from '../../../hooks/useMatrixClient';
 
@@ -38,23 +38,23 @@ export function About({ requestClose }: AboutProps) {
                 <Box shrink="No">
                   <img
                     style={{ width: toRem(60), height: toRem(60) }}
-                    src={CinnySVG}
-                    alt="Cinny logo"
+                    src={HarrierSVG}
+                    alt="Harrier logo"
                   />
                 </Box>
                 <Box direction="Column" gap="300">
                   <Box direction="Column" gap="100">
                     <Box gap="100" alignItems="End">
-                      <Text size="H3">Cinny</Text>
+                      <Text size="H3">Harrier</Text>
                       <Text size="T200">v{__CINNY_VERSION__}</Text>
                     </Box>
-                    <Text>Yet another matrix client.</Text>
+                    <Text>Powered by Cinny.</Text>
                   </Box>
 
                   <Box gap="200" wrap="Wrap">
                     <Button
                       as="a"
-                      href="https://github.com/cinnyapp/cinny"
+                      href="https://github.com/schadin/harrier-desktop"
                       rel="noreferrer noopener"
                       target="_blank"
                       variant="Secondary"
@@ -65,19 +65,6 @@ export function About({ requestClose }: AboutProps) {
                     >
                       <Text size="B300">Source Code</Text>
                     </Button>
-                    <Button
-                      as="a"
-                      href="https://cinny.in/#sponsor"
-                      rel="noreferrer noopener"
-                      target="_blank"
-                      variant="Critical"
-                      fill="Soft"
-                      size="300"
-                      radii="300"
-                      before={<Icon src={Icons.Heart} size="100" filled />}
-                    >
-                      <Text size="B300">Support</Text>
-                    </Button>
                   </Box>
                 </Box>
               </Box>
@@ -124,6 +111,35 @@ export function About({ requestClose }: AboutProps) {
                       paddingLeft: config.space.S400,
                     }}
                   >
+                    <li>
+                      <Text size="T300">
+                        Based on{' '}
+                        <a
+                          href="https://github.com/cinnyapp/cinny"
+                          rel="noreferrer noopener"
+                          target="_blank"
+                        >
+                          Cinny
+                        </a>{' '}
+                        ©{' '}
+                        <a
+                          href="https://github.com/ajbura"
+                          rel="noreferrer noopener"
+                          target="_blank"
+                        >
+                          Ajay Bura
+                        </a>{' '}
+                        used under the terms of{' '}
+                        <a
+                          href="https://github.com/cinnyapp/cinny/blob/dev/LICENSE"
+                          rel="noreferrer noopener"
+                          target="_blank"
+                        >
+                          AGPL-3.0
+                        </a>
+                        .
+                      </Text>
+                    </li>
                     <li>
                       <Text size="T300">
                         The{' '}
diff --git i/src/app/features/settings/devices/LocalBackup.tsx w/src/app/features/settings/devices/LocalBackup.tsx
index 00128c8..c1d3906 100644
--- i/src/app/features/settings/devices/LocalBackup.tsx
+++ w/src/app/features/settings/devices/LocalBackup.tsx
@@ -28,7 +28,7 @@ function ExportKeys() {
         const blob = new Blob([encKeys], {
           type: 'text/plain;charset=us-ascii',
         });
-        FileSaver.saveAs(blob, 'cinny-keys.txt');
+        FileSaver.saveAs(blob, 'harrier-keys.txt');
       },
       [mx]
     )
diff --git i/src/app/features/settings/notifications/SystemNotification.tsx w/src/app/features/settings/notifications/SystemNotification.tsx
index ebd12d5..b11fdc2 100644
--- i/src/app/features/settings/notifications/SystemNotification.tsx
+++ w/src/app/features/settings/notifications/SystemNotification.tsx
@@ -28,7 +28,7 @@ function EmailNotification() {
             device_display_name: email,
             lang: 'en',
             data: {
-              brand: 'Cinny',
+              brand: 'Harrier',
             },
             append: true,
           });
diff --git i/src/app/pages/auth/AuthFooter.tsx w/src/app/pages/auth/AuthFooter.tsx
index edfce89..bca7662 100644
--- i/src/app/pages/auth/AuthFooter.tsx
+++ w/src/app/pages/auth/AuthFooter.tsx
@@ -5,20 +5,17 @@ import * as css from './styles.css';
 export function AuthFooter() {
   return (
     <Box className={css.AuthFooter} justifyContent="Center" gap="400" wrap="Wrap">
-      <Text as="a" size="T300" href="https://cinny.in" target="_blank" rel="noreferrer">
-        About
-      </Text>
       <Text
         as="a"
         size="T300"
-        href="https://github.com/ajbura/cinny/releases"
+        href="https://github.com/schadin/harrier-desktop"
         target="_blank"
         rel="noreferrer"
       >
-        v{__CINNY_VERSION__}
+        About
       </Text>
-      <Text as="a" size="T300" href="https://twitter.com/cinnyapp" target="_blank" rel="noreferrer">
-        Twitter
+      <Text as="span" size="T300">
+        v{__CINNY_VERSION__}
       </Text>
       <Text as="a" size="T300" href="https://matrix.org" target="_blank" rel="noreferrer">
         Powered by Matrix
diff --git i/src/app/pages/auth/AuthLayout.tsx w/src/app/pages/auth/AuthLayout.tsx
index 3943f42..7e78560 100644
--- i/src/app/pages/auth/AuthLayout.tsx
+++ w/src/app/pages/auth/AuthLayout.tsx
@@ -20,7 +20,7 @@ import {
 } from '../../hooks/useClientConfig';
 import { AsyncStatus, useAsyncCallback } from '../../hooks/useAsyncCallback';
 import { LOGIN_PATH, REGISTER_PATH, RESET_PASSWORD_PATH } from '../paths';
-import CinnySVG from '../../../../public/res/svg/cinny.svg';
+import HarrierSVG from '../../../../public/res/svg/harrier.svg';
 import { ServerPicker } from './ServerPicker';
 import { AutoDiscoveryAction, autoDiscovery } from '../../cs-api';
 import { SpecVersionsLoader } from '../../components/SpecVersionsLoader';
@@ -134,8 +134,8 @@ export function AuthLayout() {
         <Box direction="Column" className={css.AuthCard}>
           <Header className={css.AuthHeader} size="600" variant="Surface">
             <Box grow="Yes" direction="Row" gap="300" alignItems="Center">
-              <img className={css.AuthLogo} src={CinnySVG} alt="Cinny Logo" />
-              <Text size="H3">Cinny</Text>
+              <img className={css.AuthLogo} src={HarrierSVG} alt="Harrier Logo" />
+              <Text size="H3">Harrier</Text>
             </Box>
           </Header>
           <Box className={css.AuthCardContent} direction="Column">
diff --git i/src/app/pages/auth/login/PasswordLoginForm.tsx w/src/app/pages/auth/login/PasswordLoginForm.tsx
index 62f46dd..8e62284 100644
--- i/src/app/pages/auth/login/PasswordLoginForm.tsx
+++ w/src/app/pages/auth/login/PasswordLoginForm.tsx
@@ -133,7 +133,7 @@ export function PasswordLoginForm({ defaultUsername, defaultEmail }: PasswordLog
         user: username,
       },
       password,
-      initial_device_display_name: 'Cinny Web',
+      initial_device_display_name: 'Harrier',
     });
   };
 
@@ -151,7 +151,7 @@ export function PasswordLoginForm({ defaultUsername, defaultEmail }: PasswordLog
         user: mxIdUsername,
       },
       password,
-      initial_device_display_name: 'Cinny Web',
+      initial_device_display_name: 'Harrier',
     });
   };
   const handleEmailLogin = (email: string, password: string) => {
@@ -163,7 +163,7 @@ export function PasswordLoginForm({ defaultUsername, defaultEmail }: PasswordLog
         address: email,
       },
       password,
-      initial_device_display_name: 'Cinny Web',
+      initial_device_display_name: 'Harrier',
     });
   };
 
diff --git i/src/app/pages/auth/login/TokenLogin.tsx w/src/app/pages/auth/login/TokenLogin.tsx
index 761d5dc..86e20ea 100644
--- i/src/app/pages/auth/login/TokenLogin.tsx
+++ w/src/app/pages/auth/login/TokenLogin.tsx
@@ -57,7 +57,7 @@ export function TokenLogin({ token }: TokenLoginProps) {
     startLogin(baseUrl, {
       type: 'm.login.token',
       token,
-      initial_device_display_name: 'Cinny Web',
+      initial_device_display_name: 'Harrier',
     });
   }, [baseUrl, token, startLogin]);
 
diff --git i/src/app/pages/auth/register/PasswordRegisterForm.tsx w/src/app/pages/auth/register/PasswordRegisterForm.tsx
index 9f17342..cbc628f 100644
--- i/src/app/pages/auth/register/PasswordRegisterForm.tsx
+++ w/src/app/pages/auth/register/PasswordRegisterForm.tsx
@@ -109,7 +109,7 @@ function RegisterUIAFlow({
         auth: authDict,
         password,
         username,
-        initial_device_display_name: 'Cinny Web',
+        initial_device_display_name: 'Harrier',
       });
     },
     [onRegister, formData]
@@ -250,7 +250,7 @@ export function PasswordRegisterForm({
       auth: {
         session: authData.session,
       },
-      initial_device_display_name: 'Cinny Web',
+      initial_device_display_name: 'Harrier',
     });
   };
 
diff --git i/src/app/pages/client/ClientNonUIFeatures.tsx w/src/app/pages/client/ClientNonUIFeatures.tsx
index 5e7dadf..149b2ea 100644
--- i/src/app/pages/client/ClientNonUIFeatures.tsx
+++ w/src/app/pages/client/ClientNonUIFeatures.tsx
@@ -11,9 +11,9 @@ import {
   RoomEventHandlerMap,
 } from 'matrix-js-sdk';
 import { roomToUnreadAtom, unreadEqual, unreadInfoToUnread } from '../../state/room/roomToUnread';
-import LogoSVG from '../../../../public/res/svg/cinny.svg';
-import LogoUnreadSVG from '../../../../public/res/svg/cinny-unread.svg';
-import LogoHighlightSVG from '../../../../public/res/svg/cinny-highlight.svg';
+import LogoSVG from '../../../../public/res/svg/harrier.svg';
+import LogoUnreadSVG from '../../../../public/res/svg/harrier-unread.svg';
+import LogoHighlightSVG from '../../../../public/res/svg/harrier-highlight.svg';
 import NotificationSound from '../../../../public/sound/notification.ogg';
 import InviteSound from '../../../../public/sound/invite.ogg';
 import { setFavicon } from '../../utils/dom';
diff --git i/src/app/pages/client/WelcomePage.tsx w/src/app/pages/client/WelcomePage.tsx
index 937cfd2..b54d005 100644
--- i/src/app/pages/client/WelcomePage.tsx
+++ w/src/app/pages/client/WelcomePage.tsx
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Box, Button, Icon, Icons, Text, config, toRem } from 'folds';
 import { Page, PageHero, PageHeroSection } from '../../components/page';
-import CinnySVG from '../../../../public/res/svg/cinny.svg';
+import HarrierSVG from '../../../../public/res/svg/harrier.svg';
 
 export function WelcomePage() {
   return (
@@ -14,18 +14,11 @@ export function WelcomePage() {
       >
         <PageHeroSection>
           <PageHero
-            icon={<img width="70" height="70" src={CinnySVG} alt="Cinny Logo" />}
-            title="Welcome to Cinny"
+            icon={<img width="70" height="70" src={HarrierSVG} alt="Harrier Logo" />}
+            title="Welcome to Harrier"
             subTitle={
               <span>
-                Yet another matrix client.{' '}
-                <a
-                  href="https://github.com/cinnyapp/cinny/releases"
-                  target="_blank"
-                  rel="noreferrer noopener"
-                >
-                  v{__CINNY_VERSION__}
-                </a>
+                Powered by Cinny. v{__CINNY_VERSION__}
               </span>
             }
           >
@@ -33,7 +26,7 @@ export function WelcomePage() {
               <Box grow="Yes" style={{ maxWidth: toRem(300) }} direction="Column" gap="300">
                 <Button
                   as="a"
-                  href="https://github.com/cinnyapp/cinny"
+                  href="https://github.com/schadin/harrier-desktop"
                   target="_blank"
                   rel="noreferrer noopener"
                   before={<Icon size="200" src={Icons.Code} />}
@@ -42,18 +35,6 @@ export function WelcomePage() {
                     Source Code
                   </Text>
                 </Button>
-                <Button
-                  as="a"
-                  href="https://cinny.in/#sponsor"
-                  target="_blank"
-                  rel="noreferrer noopener"
-                  fill="Soft"
-                  before={<Icon size="200" src={Icons.Heart} />}
-                >
-                  <Text as="span" size="B400" truncate>
-                    Support
-                  </Text>
-                </Button>
               </Box>
             </Box>
           </PageHero>
